src/core/QCheck_ng.ml

```diff
@@ -0,0 +1,1966 @@
+(*
+QCheck: Random testing for OCaml
+copyright (c) 2013-2017, Guillaume Bury, Simon Cruanes, Vincent Hugot, Jan Midtgaard
+all rights reserved.
+*)
+
+(** {1 Quickcheck inspired property-based testing} *)
+
+let poly_compare=compare
+open Printf
+
+module RS = Random.State
+
+let (|>) x f = f x
```

Comment:
The minimum version of OCaml to install QCheck is `4.03` and `|>` was introduced in `4.01`

Reply:
I removed it, thank you!